www/src/components/navigation/moreMenu.astro

```diff
@@ -45,7 +47,43 @@ const isRtl = getIsRtlFromUrl(pathname);
             ></path>
           </svg>
         </span>
-        <span>Edit this page</span>
+        <span> Edit this page</span>
+      </a>
+    </li>
+
+    <li
+      class={clsx(
+        "hidden text-t3-purple-800 hover:text-t3-purple-400 dark:text-t3-purple-200 dark:hover:text-t3-purple-100",
+        isLangEN ? ["md:block"] : [""],
```

Comment:
Logic is reversed, currently shows on english pages but not the others

```suggestion
        !isLangEN && "md:block",
```

Reply:
> Logic is reversed

It would make more sense to show the "translate" button on English pages. This way, users who are viewing the English version of the page will have the option to easily translate it if they need to. Additionally, showing the "translate" button on already translated pages may not be necessary, as the page is already in the desired language. Instead, we could consider adding a link to the "TRANSLATIONS.md" file in the updated banner section, which would provide users with a way to access the translation information and potentially contribute to the translations themselves. This way, we can ensure that the translation process is transparent and accessible to those who are interested in updating the translations.

What do you think of this approach @juliusmarminge?

Also, refer:

- EN

![image](https://user-images.githubusercontent.com/89210438/206850789-c66c16ed-a87b-42cb-8d35-35bbadec543f.png)

- BR

![image](https://user-images.githubusercontent.com/89210438/206850835-dff46389-2df9-4694-ad22-684e2406f12e.png)